eth/backend.go

```diff
@@ -590,17 +590,21 @@ func (s *Ethereum) stopAnnounce() error {
 	return nil
 }
 
-func (s *Ethereum) StartProxyHandler() error {
+func (s *Ethereum) StartProxiedValidatorEngine() error {
 	if istanbul, ok := s.engine.(consensus.Istanbul); ok {
-		return istanbul.StartProxyHandler()
+		if istanbul.IsProxiedValidator() {
+			return istanbul.StartProxiedValidatorEngine()
+		}
 	}
 
 	return nil
 }
 
-func (s *Ethereum) StopProxyHandler() error {
+func (s *Ethereum) StopProxiedValidatorEngine() error {
 	if istanbul, ok := s.engine.(consensus.Istanbul); ok {
-		return istanbul.StopProxyHandler()
+		if istanbul.IsProxiedValidator() {
```

Comment:
should we return an error if we're trying to stop when not a proxied validator?
Always better to  "fail fast"

Reply:
(i notice we call it always on `Stop`) so maybe it's about changing method name "StopIf..." 

btw, is there any other callers? maybe this does not need to be public